lib/abroad/extractors/xml/xml_extractor.rb

```diff
@@ -16,10 +16,8 @@ def extract_each(options = {}, &block)
         private
 
         def parse
-          Nokogiri::XML(stream) do |config|
-            # don't allow network connections
-            config.options = Nokogiri::XML::ParseOptions::NONET
-          end
+          options = Nokogiri::XML::ParseOptions.new.default_xml.nonet.noent
```

Comment:
Hehehehe good question. `nonet` disables network calls when parsing XML (the XML standard apparently provides for fetching remote XML resources during parse, who knew). `noent` means don't mess with HTML entities, although it doesn't appear to have any affect. I'll remove it.